stable/weave-cloud/templates/deployment.yaml

```diff
@@ -13,30 +13,30 @@ spec:
   revisionHistoryLimit: 2
   selector:
     matchLabels:
-      app: {{ .Values.agent.name }}
+      app: {{ template "weave-cloud.name" . }}
       release: {{ .Release.Name }}
   template:
     metadata:
       labels:
-        app: {{ .Values.agent.name }}
+        app: {{ template "weave-cloud.name" . }}
         release: {{ .Release.Name }}
     spec:
       {{- if .Values.serviceAccount.create }}
-      serviceAccount: {{ .Values.agent.name }}
-      serviceAccountName: {{ .Values.agent.name }}
+      serviceAccount: {{ template "weave-cloud.serviceAccountName" . }}
```

Comment:
You shouldn't need this. It's deprecated.

Reply:
I've removed it. Thanks